src/pages/PageHome.vue

```diff
@@ -46,22 +46,25 @@
         </div>
         <h2>
           <a
-            href="https://discourse.mozilla.org/t/dinopark-faq/33564"
+            href="https://discourse.mozilla.org/t/profile-privacy-display-levels-explained/37906"
             target="_blank"
             >Your privacy is protected</a
           >
         </h2>
         <p>
-          Your profile data is fully hidden from public view. Read our FAQ for
-          details.
+          Your profile data is set to Staff-only visibility by default. You can control this via field-level privacy settings. Read our <a
```

Comment:
Should we explain the _naded_ exceptions here?

Reply:
I added a commit with the precise wording we had agreed on. Hope this can be merged and deployed now.